.circleci/config.yml

```diff
@@ -0,0 +1,74 @@
+version: 2
```

Comment:
Please copy the new version from the CLI and remove just the integration tests.

Reply:
Done